README.md

```diff
@@ -13,6 +13,8 @@ git clone https://github.com/NYPL-Simplified/registry_admin.git
 
 ## Installation (Docker)
 
+If not using Docker, skip to section entitled "Installation (non-Docker)."
```

Comment:
Not necessary but you can link to the heading by doing `["Installation (non-Docker)"](#installation-non-docker).`.

Reply:
Cool! Fixed!